accounts/views.py

```diff
@@ -73,7 +76,31 @@ def get(self, request):
             'detail': serializer.errors
         }, status=status.HTTP_400_BAD_REQUEST)
             
-            
+
+# user view
+class UserDetailView(RetrieveAPIView):
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
```

Comment:
1. Created a class UserDetailView that inherits from RetrieveAPIView. RetrieveAPIView is a built-in view from Django REST framework that handles retrieving model's data.
2. The second line specifies that the view will retrieve data from the User model's data using User.objects.all(), which returns a queryset of all User model's data in the database.
3. The third line identifies UserSerializer as the serializer class that will be used to convert User model's data to and from JSON format.